docs/azureai/azureai-search-document-integration.md

```diff
@@ -1,22 +1,154 @@
 ---
-title: .NET Aspire Azure AI Search Documents integration
-description: Learn how to use the .NET Aspire Azure AI Search Documents integration.
-ms.topic: how-to
-ms.date: 08/12/2024
+title: .NET Aspire Azure AI Search integration
+description: Learn how to integrate Azure AI Search with .NET Aspire.
+ms.date: 03/05/2025
 ---
 
-# .NET Aspire Azure AI Search Documents integration
+# .NET Aspire Azure AI Search integration
 
-In this article, you learn how to use the .NET Aspire Azure AI Search Documents client. The `Aspire.Azure.Search.Documents` library is used to register an <xref:Azure.Search.Documents.Indexes.SearchIndexClient> in the dependency injection (DI) container for connecting to Azure Search. It enables corresponding health checks and logging.
+[!INCLUDE [includes-hosting-and-client](../includes/includes-hosting-and-client.md)]
 
-For more information on using the `SearchIndexClient`, see [How to use Azure.Search.Documents in a C# .NET Application](/azure/search/search-howto-dotnet-sdk).
+[Azure AI Search](/azure/search/search-what-is-azure-search) (formerly Azure Cognitive Search) is an enterprise-ready information retrieval system for your heterogeneous content that you ingest into a search index, and surface to users through queries and apps. It comes with a comprehensive set of advanced search technologies, built for high-performance applications at any scale. The .NET Aspire Azure AI Search Documents integration enables you to connect to Azure AI Search services from your .NET applications.
```

Comment:
That first sentence is full of jargon. I suggest starting with the simpler statement, then clarifying with the jargon.

```suggestion
The .NET Aspire Azure AI Search Documents integration enables you to connect to [Azure AI Search](/azure/search/search-what-is-azure-search) (formerly Azure Cognitive Search) services from your .NET applications. Azure AI Search is an enterprise-ready information retrieval system for your heterogeneous content that you ingest into a search index, and surface to users through queries and apps. It comes with a comprehensive set of advanced search technologies, built for high-performance applications at any scale.
```